### ISSUE TITLE
[Help Wanted] Update password reset page

### ISSUE THREAD

**Issue by jasonblais** [mattermost/platform#4497]
If you're interested please comment here and come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.

---

**Details** ([Jira ticket](https://mattermost.atlassian.net/browse/PLT-4524))

Update password reset page

Observed: 
1. Click "I forgot my password"
2. Enter an email that doesn't have an account on the server
3. An error message "We couldn't find an account with that address." is shown

Proposed change: 
Instead of showing the error message, take them to the page that is shown when the reset password email is sent (see screenshot for current version below). Update the text on that page to say:

```
If the account exists, a password reset email will be sent to: 
[email address]

Please check your inbox.
```

![image](https://cloud.githubusercontent.com/assets/13119842/20106406/07810cc6-a5a3-11e6-9497-55cdedcdf020.png)

**Comment by cpanato** [comment]
Will work on that, is that ok?

**Comment by cpanato** [comment]
PR #4522

**Comment by jasonblais** [comment]
Closed via #4522
